client/ayon_substancepainter/plugins/create/create_textures.py

```diff
@@ -142,12 +142,25 @@ def get_instance_attr_defs(self):
                 "CoatSpecularLevel": "Coat Specular Level",
                 "CoatNormal": "Coat Normal",
             }
+        export_texture_set_enum = {
+            texture_set.name(): texture_set.name()
+            for texture_set in substance_painter.textureset.all_texture_sets()
+        }
 
         return [
             BoolDef("review",
                     label="Review",
                     tooltip="Mark as reviewable",
                     default=True),
+            EnumDef("exportTextureSet",
```

Comment:
The result is a list of things, so should really be plural?
```suggestion
            EnumDef("exportTextureSets",
```